src/collector/api_server.py

```diff
@@ -102,3 +103,40 @@ def collect(self):
               for m in metrics.values()]
       self._db_con.write_data(data)
       time.sleep(wait_time)
+
+  def add_topic(self):
+    # On the situation where a new solar panel is introduced, this method
+    # adds the corresponding topic names to the database.
```

Comment:
This comment is inaccurate. This CL doesn't automatically add the topic name to the database. This just adds an endpoint.

Also, the rest of the methods use pydoc. Please update the format of this comment.